### DISCUSSION TITLE
Volt

### DISCUSSION THREAD

**Discussion by T3mmyKay** [mhmiton/laravel-modules-livewire#44] [Ideas]
Will there be a support for volt

**Comment by mhmiton** [comment]
@T3mmyKay Sure, I will try to work with this.

**Reply by OccultScientist** [reply]
I'd be really interested in this too!
Any updates?

**Comment by mhmiton** [comment]
@T3mmyKay @OccultScientist The volt support is added. Please check version 4.

https://github.com/mhmiton/laravel-modules-livewire/releases/tag/v4.0.0
